index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <img
+        id="header-img"
+        src="https://graffica.info/wp-content/uploads/2022/04/logotipo-nike-838x285x80xX.jpeg"
+        alt="" />
+
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#productinfo">Product Info</a></li>
+          <li><a class="nav-link" href="#prices">Prices</a></li>
+          <li><a class="nav-link" href="#video">Watch the Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="productinfo">
+        <p>This is "productinfo" section</p>
+      </section>
+
+      <section id="prices">
+        <p>This is "prices" section</p>
+      </section>
+
+      <section id="watchvideo">
+        <p>Click play to watch the video</p>
+        <iframe
+          id="video"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/MbjBFMKT6Xs"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+      </section>
+
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="get">
+        <input
```

Comment:
You should use the post method for security purposes.

Reply:
- [ ] Kindly make sure your comments are given in a polite and friendly way :smile: They should not sound like orders but suggestions.
- [ ] Consider explaining why the changes are suggested as well. You may need to google it and see exactly what the changes you suggested help improve in the project.
As here `POST` method is not for security purposes :face_with_spiral_eyes:

Reply:
I think it would be better to use the post method. Because POST is a little safer than GET because the parameters are not stored in browser history or in web server logs.

Reply:
- [Optional] When you are facing this kind of situation where the explanation can be large or complex you can complement your comment by adding a link to a page that describes the topic more deeply. 🤓  